docs/admin/authorization/rbac.md

```diff
@@ -151,19 +162,80 @@ metadata:
   namespace: default
   name: pod-and-pod-logs-reader
 rules:
-  - apiGroups: [""]
-    resources: ["pods", "pods/log"]
-    verbs: ["get", "list"]
+- apiGroups: [""]
+  resources: ["pods", "pods/log"]
+  verbs: ["get", "list"]
+```
+
+#### Role examples
+
+Only the `rules` section is shown in the following examples.
+
+Allow reading the resource "pods" in the core API group:
+
+```yaml
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+```
+
+Allow reading/writing "deployments" in both the "extensions" and "apps" API groups:
+
+```yaml
+rules:
+- apiGroups: ["extensions", "apps"]
+  resources: ["deployments"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
 ```
 
-## Referring to Subjects
+Allow reading "pods" and reading/writing "jobs":
 
-RoleBindings and ClusterRoleBindings bind "subjects" to "roles".
+```yaml
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["batch", "extensions"]
+  resources: ["jobs"]
+  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
+```
+
+Allow reading a configmap named "my-config" (must be bound with a `RoleBinding` to limit to a single configmap in a single namespace):
```

Comment:
- [ ] Do we usually write "configmap" in all lowercase?  
  We could also write it as ConfigMap, or _ConfigMap_, but I'm not sure if there's a convention.